From 6c9c6b122b5109c949d31a0a3ce91cfc2fc9879e Mon Sep 17 00:00:00 2001
From: Tomas Winkler <tomas.winkler@intel.com>
Date: Tue, 18 Mar 2014 22:52:07 +0200
Subject: [PATCH 343/441] mei: me: bump hbm version to 1.1 to support power
 gating

Communicate hbm version 1.1 to firmware to tell that we
now support power gating isolation protocol

Change-Id: I948bd53ec3fe625d8d25e9842572ae7ba98d5821
Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
(cherry picked from commit 2036822476b43efba72429badcbbc9e56c0ce5dd)

Signed-off-by: Nagaraj S <sivasankaranx.nagaraj@intel.com>
---
 drivers/misc/mei/hw.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/misc/mei/hw.h
+++ b/drivers/misc/mei/hw.h
@@ -37,7 +37,7 @@
 /*
  * MEI Version
  */
-#define HBM_MINOR_VERSION                   0
+#define HBM_MINOR_VERSION                   1
 #define HBM_MAJOR_VERSION                   1
 
 /*
